### PR TITLE
[Term Entry] PyTorch Tensors .eye()

### LINKED ISSUE #5302: [Term Entry] PyTorch Tensors .eye()
### Reason/inspiration (optional)

We would like a new entry on the `.eye()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensors/terms/eye/eye.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by PragatiVerma18**
### Description

Add a new entry on the `.eye()` term under PyTorch.

### Issue Solved

Closes #5302

### Type of Change

- [x] Adding a new entry
- [ ] Editing an existing entry (fixing a typo, bug, issues, etc)
- [ ] Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensors/terms/eye/eye.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+The **`.eye()`** function in PyTorch is used to create a 2D tensor representing an identity matrix. An identity matrix is a square matrix in which all elements of the principal diagonal are ones, and all other elements are zeros.
+
+This function is commonly used in linear algebra operations and initializing neural network weights.
+
+## Syntax
+
+```pseudo
+torch.eye(n, m, dtype=None, device=None, out=None)
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/eye/eye.md:23]: Let us include the entire syntax as follows:

```pseudo
torch.eye(n, m=None, *, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False)
```

File: content/pytorch/concepts/tensors/terms/eye/eye.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+- `n`: The number of rows in the resulting identity matrix.
+- `m`: The number of columns in the resulting identity matrix.
+- `dtype`: The desired data type of the returned tensor (optional).
+- `device`: The device on which the tensor should be allocated (optional).
+- `out`: The output tensor (optional).
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/eye/eye.md:30]: Please add the rest of the parameter descriptions as well. And make sure that the 'optional' is on the left side, example:

- `out` (Optional): The output tensor.

File: content/pytorch/concepts/tensors/terms/eye/eye.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Example
+
+Here is an example that demonstrates the use of `.eye()` to create a 3x3 identity matrix:
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/eye/eye.md:34]:
```suggestion
Here is an example that demonstrates the use of `.eye()` to create a *3x3* identity matrix:
```

File: content/pytorch/concepts/tensors/terms/eye/eye.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+```shell
+tensor([[1., 0., 0.],
+        [0., 1., 0.],
+        [0., 0., 1.]])
+```
+
+## Codebyte Example
+
+Here is an example that demonstrates the use of `.eye()` to create a 4x4 identity matrix:
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/eye/eye.md:55]:
```suggestion
Here is an example that demonstrates the use of `.eye()` to create a *4x4* identity matrix:
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @PragatiVerma18 , thank you for contributing to Codecademy Docs, the entry is nicely written! 😄 

I've suggested a few changes, could you please review and modify those at your earliest convenience? Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @PragatiVerma18  😄 

The entry looks good to be merged! 🚀
